openpdf/pom.xml

```diff
@@ -34,6 +34,13 @@
             <artifactId>bcpkix-jdk15on</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
probably want the annotations too? https://mvnrepository.com/artifact/com.github.spotbugs/spotbugs-annotations/3.1.12

Reply:
It's already transitive